tests/load-tests/pkg/journey/handle_integration_test_scenarios.go

```diff
@@ -40,20 +40,12 @@ func validateIntegrationTestScenario(f *framework.Framework, namespace, name, ap
 			return false, nil
 		}
 
-		// Check if there are some conditions
-		if len(its.Status.Conditions) == 0 {
-			logging.Logger.Debug("Integration test scenario %s for application %s in namespace %s lacks status conditions", name, appName, namespace)
-			return false, nil
-		}
-
 		// Check right condition status
 		for _, condition := range its.Status.Conditions {
 			if (strings.HasPrefix(condition.Type, "Error") || strings.HasSuffix(condition.Type, "Error")) && condition.Status == "True" {
 				return false, fmt.Errorf("Integration test scenario %s for application %s in namespace %s is in error state: %+v", name, appName, namespace, condition)
 			}
-			if condition.Type == "IntegrationTestScenarioValid" && condition.Status == "True" {
-				return true, nil
```

Comment:
Hello @larrymou9 . Looking at the patch, I think with this change we would never get out of `WaitUntilWithInterval`-loop. Have you tested the change?